### ISSUE TITLE
wano act 2 doesnt work

### ISSUE THREAD

**Issue by 0bscenity** [jwueller/jellyfin-plugin-onepace#66]

**Comment by jwueller** [comment]
Hi @0bscenity! Which version of Jellyfin and the plugin are you currently running?

**Comment by 0bscenity** [comment]
> Hi @0bscenity! Which version of Jellyfin and the plugin are you currently running?

Hi, sorry it took me so long to reply, been off github, I am now running 
```
Server version
10.9.8
Web version
10.9.8
Build version
10.9.8
```
I this is now an issue with me with any files that don't come in a "pack"

![image](https://github.com/user-attachments/assets/041e8da6-8e82-48b9-ac6d-83c9b301d5e3)
![image](https://github.com/user-attachments/assets/b5c4a899-0591-4a08-8d56-a32a513b523a)
![image](https://github.com/user-attachments/assets/a0a1f01f-fb9e-4d6a-acf7-c8ad04a1eb2b)

Here is my folder structure
```
/media/tv/One Pace/
└── [One Pace] Egghead
    ├── [One Pace][1058-1059] Egghead 01 [720p][En Sub][C42415FD].mp4
    ├── [One Pace][1059-1060] Egghead 02 [1080p][01B6EE9A].mkv
    ├── [One Pace][1060-1061] Egghead 03 [720p][En Sub][812BEB17].mp4
    ├── [One Pace][1062-1063] Egghead 04 [1080p][92F9FD5D].mkv
    ├── [One Pace][1064-1065] Egghead 05 [1080p][E80E4B80].mkv
    ├── [One Pace][1066] Egghead 06 [1080p][C9D76388].mkv
    ├── [One Pace][1067-1068] Egghead 07 [1080p][A7AD080A].mkv
    ├── [One Pace][1069-1070] Egghead 08 [1080p][126ECF70].mkv
    ├── [One Pace][1071-1072] Egghead 09 [1080p][ED808618].mkv
    ├── [One Pace][1073-1074] Egghead 10 [1080p][65405475].mkv
    └── [One Pace][1075-1076] Egghead 11 [1080p][59087997].mkv
```
I have disabled every metadata extension except for One Pace
![image](https://github.com/user-attachments/assets/c27af3ed-bb33-4948-a774-58d83d85c6ed)
![image](https://github.com/user-attachments/assets/0ed1ea44-d0a0-4a15-8821-ddd4c428cfde)
![image](https://github.com/user-attachments/assets/aa01c05e-6ba3-4b94-8fc3-211f836f04c7)

**Comment by apocaliss92** [comment]
I have exactly the same issue, since there is nothing to do until the official website will be up, would it be possible, at least, to give the single episodes a meaningful name in case of issues? The season/arc name is ok, the title is inside and it's understundable, but all the episodes are named as [One Pace and nothing else

**Comment by jwueller** [comment]
@apocaliss92 You're right, it should probably at least retain the filename. I'll try to improve dealing with the API being down a bit better.

**Comment by XadRav** [comment]
This there any update on this?  Any work around since the website still appears to be down?

**Comment by michaelBelsanti** [comment]
You can use the metadata provided in #78 or rename the episodes and arcs manually

**Comment by Ashpan** [comment]
The site has been rebuilt and is up and running again
https://onepace.net/

From the discord
> We have a new website! It's still under heavy construction, but we have some exciting features and improvements in the pipeline!

**Comment by cdgiguere** [comment]
So the website is back up, but has the plugin been fixed? I am still having this issue.
